packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2663,7 +2663,7 @@ class InputDecoration {
   /// When the [InputDecoration.labelText] is on top of the input field, the
   /// text uses the [labelStyle] instead.
   ///
-  /// If [floatingLabelStyle] is a [MaterialStateTextStyle], then the effective
+  /// If `floatingLabelStyle` is a [MaterialStateTextStyle], then the effective
```

Comment:
Is this is common pattern to use quotes for the currently described parameter instead of the link?

Reply:
Yes. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements

Reply:
TIL, thx!